pkg/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -5,44 +5,37 @@
     <BaseLinePackageDependencies>false</BaseLinePackageDependencies>
     <PackagePlatforms>x64;</PackagePlatforms>
     <PreventImplementationReference>true</PreventImplementationReference>
+    <RestorePackages>true</RestorePackages>
     <SkipValidatePackage>true</SkipValidatePackage>
     <IncludeSymbolsInPackage Condition="'$(IncludeSymbolsInPackage)' == ''">true</IncludeSymbolsInPackage>
-
+    <!-- Mark this package as a lineup (meta-) package -->
+    <IsLineupPackage Condition="'$(IsLineupPackage)' == ''">true</IsLineupPackage>
     <!-- Override this property so that the package name won't look like runtime.[RID].[TFM].[ID] -->
     <PackageTargetRuntime></PackageTargetRuntime>
   </PropertyGroup>
+
   <ItemGroup>
-    <File Include="$(MSBuildThisFileDirectory)\Microsoft.DotNet.ILCompiler.targets" >
+    <ProjectReference Include="win\Microsoft.DotNet.ILCompiler.pkgproj" />
```

Comment:
Do we want to set this up such that it works for win-x64, win-x86, linux-arm, ... ?

I thought these runtime specific packages can be structured such that the metapackage does not need to know about every runtime specific package.

Reply:
That actually seems correct. 
I made the wrong assumption that the metapackage's runtime.json was generated from its `.pkgproj`, but this isn't the case. As long as the runtime-specific packages have a dedicate package definition in a sub-folder, this is done automatically.
The current setup, however does only build packages according to which machine we're running on. Is cross-building of packages something we're looking into supporting?

Reply:
Eventually - yes.

I expect that the eventual CoreRT platform matrix would be same as CoreCLR. So the configs where we cross-build for CoreCLR like linux-arm would cross build for CoreRT as well.